### PR TITLE
fix report field type changed

### PR DESCRIPTION
**Pull request by lideke**

### CODE REVIEW

File: core/src/test/java/io/anemos/metastore/core/proto/validation/ProtoDiffTest.java [hunk 1]
```diff
@@ -98,6 +96,38 @@
                   .build())
           .build();
 
+  public static final DescriptorProtos.FileDescriptorProto WRAPPER_V1 =
```
Comment (alexvanboxel) [inline core/src/test/java/io/anemos/metastore/core/proto/validation/ProtoDiffTest.java:99]: I want you to collapse this into the FILE_V1 test descriptor, you can add 1 field to the single message and play with the wrapper type.

File: core/src/test/proto/test/v1/wrapper.proto [hunk 2]
```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
```
Comment (alexvanboxel) [inline core/src/test/proto/test/v1/wrapper.proto:1]: Remove this file, it's unnecessary. You can create the test by expanding the FILE_V1/FILE_V1ALPHA test descriptor.

File: core/src/test/java/io/anemos/metastore/core/proto/validation/ProtoDiffTest.java [hunk 3]
```diff
@@ -319,6 +349,105 @@ public void renameField() throws Exception {
...
+    Assert.assertEquals(ChangeType.UNCHANGED, result.getChange().getChangeType());
+    Assert.assertEquals(ChangeType.CHANGED, result.getFieldResults(0).getChange().getChangeType());
+    Assert.assertEquals(
+        "google.protobuf.StringValue", result.getFieldResults(0).getChange().getFromTypeName());
+    Assert.assertEquals(
+        "google.protobuf.Int64Value", result.getFieldResults(0).getChange().getToTypeName());
+  }
+
+  @Test
+  public void changeWrapperFieldTypeToPrimitive() throws Exception {
```
Comment (alexvanboxel) [inline core/src/test/java/io/anemos/metastore/core/proto/validation/ProtoDiffTest.java:386]: Wrapper is not a concept that is native to metastore, boil it down to its essence.

File: core/src/test/java/io/anemos/metastore/core/proto/validation/ProtoDiffTest.java [hunk 4]
```diff
@@ -319,6 +349,105 @@ public void renameField() throws Exception {
...
+    Assert.assertEquals(ChangeType.CHANGED, result.getFieldResults(0).getChange().getChangeType());
+    Assert.assertEquals(
+        "google.protobuf.StringValue", result.getFieldResults(0).getChange().getFromTypeName());
+    Assert.assertEquals(
+        FieldChangeInfo.FieldType.FIELD_TYPE_STRING,
+        result.getFieldResults(0).getChange().getToType());
+  }
+
+  @Test
+  public void changeNestedFieldType() throws Exception {
```
Comment (alexvanboxel) [inline core/src/test/java/io/anemos/metastore/core/proto/validation/ProtoDiffTest.java:419]: Nested name is incorrectly used as a nested message is a message in another message. If you use the original FILE descriptor you should not need to use those terminology.

File: core/src/test/java/io/anemos/metastore/core/proto/validation/ProtoDiffTest.java [hunk 5]
```diff
@@ -4,14 +4,12 @@
 import io.anemos.metastore.core.proto.validate.ProtoDiff;
 import io.anemos.metastore.core.proto.validate.ValidationResults;
 import io.anemos.metastore.putils.ProtoDomain;
-import io.anemos.metastore.v1alpha1.ChangeType;
-import io.anemos.metastore.v1alpha1.EnumResult;
-import io.anemos.metastore.v1alpha1.MessageResult;
-import io.anemos.metastore.v1alpha1.Report;
-import io.anemos.metastore.v1alpha1.ServiceResult;
+import io.anemos.metastore.v1alpha1.*;
```
[the remark this answers is not in the document]
Comment (lideke) [reply core/src/test/java/io/anemos/metastore/core/proto/validation/ProtoDiffTest.java:7]: this is spotless apply

**Pull request merged**